blockchain/tx_pool_test.go

```diff
@@ -2083,10 +2097,12 @@ func TestTransactionsPromoteFull(t *testing.T) {
 
 	pool.promoteExecutables(nil)
 
-	assert.Equal(t, pool.pending[from].Len(), 4)
-
-	for i, tx := range txs {
-		assert.True(t, reflect.DeepEqual(tx, pool.pending[from].txs.items[uint64(i)]))
+	r := assert.NotEqual(t, len(pool.pending), 0)
+	if r {
+		assert.Equal(t, pool.pending[from].Len(), 4)
+		for i, tx := range txs {
+			assert.True(t, reflect.DeepEqual(tx, pool.pending[from].txs.items[uint64(i)]))
+		}
```

Comment:
Ideally and as preceding expected, I thought `pool.pending[from].Len()` must be four. Why may it panic by your change?

Reply:
It is a trivial issue, not critical. When I test it for kip71 hardfork, It returns the error.
I just added the guard because I don't want to see the panic. 
The error fixed now. So if you think this code is unnecessary, you can delete it.

Reply:
I deleted it.